lib/collection.js

```diff
@@ -40,7 +40,11 @@ module.exports = function Collection(db, state) {
 
     aggregate: NotImplemented,
     bulkWrite: NotImplemented,
-    count: NotImplemented,
+    count: function(query, opts, cb ){                                                                                                                                                                                                    
+      this.find().toArray( function(err, docs){
+        cb(err, docs.length)
+      })
+    },
```

Comment:
It will need to handle the query, options and possible return of a Promise. Can you try out this?
```javascript
count: function() {
  var opts = find_options(arguments);
  return this.find(opts.query || {}, opts).count(opts.callback);
},
```

I'm not able to run & test this myself right now.